### PR TITLE
:tiger: Remove unnecessary POMI data

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

File: app/lib/populateCache.js [hunk 1]
```diff
@@ -5,11 +5,12 @@ const assert = require('assert');
...
   const data = fs.readFileSync(path);
-  assert.ok(data.length > 0, `No data in file: '${path}'.`);
+  assert(data.length > 0, `No data in file: '${path}'.`);
 
   const records = parse(data, { columns: true });
   console.log(`${records.length} records read...`);
 
+  assert(records[0], `No records in file: '${path}'.`);
   records.forEach((item) => {
     cache.put(item.ods_code.toUpperCase(), item);
```
Comment (neilbmclaughlin) [inline app/lib/populateCache.js:15]: I think this will put the ODS code in the cache value as well as the cache key.

File: app/middleware/gp.js [hunk 2]
```diff
@@ -97,33 +97,11 @@ function upperCaseGpId(req, res, next) {
...
-      // eslint-disable-next-line no-param-reassign
-      req.gpDetails.bookOnlineLink = `https://systmonline.tpp-uk.com/Login?PracticeId=${gpId}`;
-      break;
-    case 'NK':
-    default:
-      // eslint-disable-next-line no-param-reassign
-      req.gpDetails.bookOnlineLink = '';
-  }
+  // eslint-disable-next-line no-param-reassign
+  req.gpDetails.bookOnlineUrl = cache.get(gpId).book_online_url;
```
Comment (neilbmclaughlin) [inline app/middleware/gp.js:104]: The URL should probably be validated here or, preferably, on load from the csv file.

File: app/lib/populateCache.js [hunk 3]
```diff
@@ -5,11 +5,12 @@ const assert = require('assert');
 
 function populateCacheSync(path) {
   const data = fs.readFileSync(path);
-  assert.ok(data.length > 0, `No data in file: '${path}'.`);
+  assert(data.length > 0, `No data in file: '${path}'.`);
 
   const records = parse(data, { columns: true });
```
Comment (neilbmclaughlin) [inline app/lib/populateCache.js:10]: Should the code be explicit about what columns it expects and only use those columns in the cache. A badly written ETL process could result in more data in the cache than is needed. 

Any thoughts?

File: app/middleware/gp.js (hunk 2 above)
Comment (st3v3nhunt) [inline app/middleware/gp.js:104]: I was working on the assumption the data in the file is 'good' and whatever process generates the file handles that. I guess it wouldn't hurt to double check here.

File: app/lib/populateCache.js (hunk 1 above)
Comment (st3v3nhunt) [inline app/lib/populateCache.js:15]: Yeah it will, i'm not overly concerned about that right now.

File: app/lib/populateCache.js (hunk 3 above)
Comment (st3v3nhunt) [inline app/lib/populateCache.js:10]: Possibly but doing that means changing this (at least how it is currently working) when the columns change. I don't want to have to make that change as this was/is intended to be a generic 'load stuff into cache'.

**Pull request merged**